cellengine/utils/types.py

```diff
@@ -0,0 +1,17 @@
+from __future__ import annotations
+import sys
+from typing import Dict, List
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+TailoringDict = TypedDict(
```

Comment:
Can you call this `ApplyTailoringRes`? That will be conducive to typing other responses in the future.